.github/workflows/scale-test-egw.yaml

```diff
@@ -0,0 +1,485 @@
+
+          metadata:
+            name: ${{ steps.vars.outputs.cluster_name }}
+            region: ${{ env.eks_region }}
+            version: "${{ env.k8s_version }}"
+            tags:
+              usage: "${{ github.repository_owner }}-${{ github.event.repository.name }}"
+              owner: "${{ steps.vars.outputs.owner }}"
+
+          managedNodeGroups:
+          - name: ng-amd64-client
+            instanceTypes:
+            - m5.large
+            desiredCapacity: ${{ steps.vars.outputs.num_client_nodes }}
+            maxPodsPerNode: 100
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/egw-client: "true"
+          - name: ng-amd64-egw-node
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/egw-node: "true"
+          - name: ng-amd64-heapster
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "node.cilium.io/agent-not-ready"
+              value: "true"
+              effect: "NoExecute"
+            labels:
+              role.scaffolding/monitoring: "true"
+          - name: ng-amd64-no-cilium
+            instanceTypes:
+            - m5.large
+            desiredCapacity: 1
+            spot: false
+            privateNetworking: true
+            volumeType: "gp3"
+            volumeSize: 10
+            ssh:
+              allow: true
+              publicKeyPath: ${{ env.AWS_SSH_KEY }}
+            taints:
+            - key: "cilium.io/no-schedule"
+              value: "true"
+              effect: "NoSchedule"
+            labels:
+              cilium.io/no-schedule: "true"
+          EOF
+
+          eksctl create cluster -f ./eks-config.yaml
+
+      - name: Patch AWS-CNI to enable prefix delegation
+        run: |
+          kubectl set env daemonset aws-node -n kube-system ENABLE_PREFIX_DELEGATION=true
+          kubectl rollout restart ds/aws-node -n kube-system
+          kubectl rollout status ds/aws-node -n kube-system -w
```

Comment:
The cilium-cli is actually [patching](https://github.com/cilium/cilium/blob/pr%2Flearnitall%2Finit-egw-scale-test/cilium-cli/install/install.go#L230-L239) the daemonset during installation to evict all pods. Is this change necessary?

Reply:
Nice catch, thanks!